SocialCareCaseViewerApi/V1/Factories/EntityFactory.cs

```diff
@@ -33,6 +43,27 @@ public static Address ToDomain(this DbAddress address)
             };
         }
 
+        public static LastUpdatedDomain DbLastUpdatedToDomain(LastUpdated lastUpdated)
+        {
+            return new LastUpdatedDomain()
+            {
+                Housing = lastUpdated.Housing,
+                ContactDetails = lastUpdated.ContactDetails
```

Comment:
these are arbitrary values for the sake of the tests, yes?

we just want this to be object with arbitrary keys and datetime values

Reply:
No these are actual values. Thanks for clarifying - I will amend.